terraform-build-modules/courseBase.sh

```diff
@@ -0,0 +1,33 @@
+# Install Terraform and init config
+
+# Installs Terraform 0.13.5
+curl -O https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip
+unzip terraform_0.13.5_linux_amd64.zip -d /usr/local/bin/
+
+# Installs awscli
+apt-get install awscli -y
+
+# Clone GitHub repo
+git clone -b localstack https://github.com/hashicorp/learn-terraform-modules
+cd ~/ec2-instances-localstack
```

Comment:
I haven't tested the katacoda yet, but it's not clear to me how this directory gets created?